docs/source/blackbox/permutation_importance.rst

```diff
@@ -0,0 +1,115 @@
+noise - feature column is still there, but it no longer contains useful
+information. This method works if noise is drawn from the same
+distribution as original feature values (as otherwise estimator may
+fail). The simplest way to get such noise is to shuffle values
+for a feature, i.e. use other example's feature value - this is how
+permutation importance is computed.
+
+Model Inspection
+----------------
+
+For sklearn-compatible estimators eli5 provides
+:class:`~.PermutationImportance` wrapper; if you want to use this
+method for other estimators there is :mod:`eli5.permutation_importance` module.
+
+For example, this is how you can check feature importances of
+`sklearn.svm.SVC`_ classifier, which is not supported by eli5 directly
+when a non-linear kernel is used::
+
+    import eli5
+    from eli5.sklearn import PermutationImportance
+    from sklearn.svm import SVC
+
+    # ... load data
+
+    svc = SVC().fit(X_train, y_train)
+    perm = PermutationImportance(svc).fit(X_test, y_test)
+    eli5.show_weights(perm)
+
+.. _sklearn.svm.SVC: http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html
+
+If you don't have a separate held-out dataset, you can fit
+:class:`~.PermutationImportance` on the same data as used for
+training; this still allows to inspect the model, but doesn't show which
+features are important for generalization.
+
+Feature Selection
+-----------------
+
+This method can be useful not only for introspection, but also for
+feature selection - one can compute feature importances using
+:class:`~.PermutationImportance`, then drop unimportant features
+using e.g. sklearn's SelectFromModel_ or RFE_. In this case estimator passed
+to :class:`~.PermutationImportance` doesn't have to be fit; feature
+importances can be computed for several train/test splits and then averaged::
+
+    import eli5
+    from eli5.sklearn import PermutationImportance
+    from sklearn.svm import SVC
+    from sklearn.feature_selection import SelectFromModel
+
+    # ... load data
+
+    perm = PermutationImportance(SVC(), cv=5)
+    perm.fit(X, y)
+
+    # perm.feature_importances_ attribute is now available, it can be used
+    # for feature selection - let's e.g. select features which increase
+    # accuracy by at least 0.05:
+    sel = SelectFromModel(perm, threshold=0.05, prefit=True)
+    X_trans = sel.transform(X)
+
+    # It is possible to combine SelectFromModel and
+    # PermutationImportance directly, without fitting
+    # PermutationImportance first:
+    sel = SelectFromModel(
+        PermutationImportance(SVC(), cv=5),
+        threshold=0.05,
+    ).fit(X, y)
+    X_trans = sel.transform(X)
+
+See :class:`~.PermutationImportance` docs for more.
+
+Note that permutation importance should be used for feature selection with
+care. For example, if several features are correlated, and the estimator
+uses them all equally, permutation importance can be low for all of these
```

Comment:
But I suspect this is somewhat true of many methods that measure feature importance in a model (rather than either associations between multiple features and the target, or univariate relationships without trying to maximise classification performance with all features).